pkg/lns/chirpstack.go

```diff
@@ -349,7 +348,7 @@ func NewChirpstackClient(client BaseClient) *ChirpstackClient {
 	)
 
 	if err != nil {
-		log.Fatalf("Cannot connect to server: %s", err)
+		fmt.Printf("[lns][error] Connection error %s\n", err)
```

Comment:
This should still be a fatal error imho; if we cant setup a grpc connection to chirpstack, there is no reason to continue